aggregator_core/src/datastore.rs

```diff
@@ -99,7 +99,7 @@ macro_rules! supported_schema_versions {
 // version is seen, [`Datastore::new`] fails.
 //
 // Note that the latest supported version must be first in the list.
-supported_schema_versions!(3, 2, 1);
+supported_schema_versions!(3);
```

Comment:
I think we'll want to discuss in release notes that deploying whichever Janus version this PR goes out in is not backward compatible and that operators will want to work out a DB migration plan.

Reply:
👍🏻, but as you mentioned in https://github.com/divviup/janus-ops/issues/1257, the DB migration is actually related to https://github.com/divviup/janus/pull/2555 which is going out in this week's release. This PR will go out with the next release, once that migration has stabilized.